KFC.Red.Server/KFC.Red.DBate.WebAPI/Controllers/QuestionController.cs

```diff
@@ -9,45 +9,36 @@
 
 namespace KFC.Red.DBate.WebAPI.Controllers
 {
-    public class QuestionManagementRequest
-    {
-        [HttpBindRequired]
-        public Question Question { get; set; }
-
-        [HttpBindRequired]
-        public int QuestionID { get; set; }
-
-        [HttpBindRequired]
-        public string QuestionString { get; set; }
-    }
-
     public class QuestionController : ApiController
     {
+        public class QuestionManagementRequest
+        {
+            public Question question { get; set; }
+            public int QuestionID { get; set; }
+            public string QuestionString { get; set; }
+        }
+
         // http GET/POST/PUT/DELETE methods based on add/update/delete questions
-        // see user controller in SSO
         [HttpPost]
         [Route("api/question/add")]
         public IHttpActionResult AddQuestion([FromBody] QuestionManagementRequest Request)
         {
             QuestionManager questionM = new QuestionManager();
-            //Question question = new Question();
             if (questionM.ExistingQuestion(Request.QuestionString) == true)
             {
                 return Content(HttpStatusCode.Conflict, "Question already exists");
             }
-            
-            // add question to database
+            questionM.CreateQuestion(Request.question);
         }
 
         [HttpPut]
         [Route("api/question/update")]
         public IHttpActionResult UpdateQuestion([FromBody] QuestionManagementRequest Request)
         {
             QuestionManager questionM = new QuestionManager();
-
             if (questionM.ExistingQuestion(Request.QuestionString) == true)
```

Comment:
In all the methods that have if/else statements that do the business logic, return ok status after finishing because it lets the system/user know it was successful.